citations-tool/tool/src/webapp/vm/citation/create.vm

```diff
@@ -376,14 +376,15 @@
 	 */
 	function trackPreferredUrl( checkboxId )
 	{
-		// has this been clicked?
-		if( $( "#" + checkboxId ).attr( "checked" ) != null )
+		var checkBoxClicked = $( "#" + checkboxId );
+		// has this been checked?
+		if( checkBoxClicked.attr( "checked" ) == true )
```

Comment:
I was going to write that as we upgrade the jquery this won't work.Yes the write way is to use element.is(:checked) or element.prop( "checked" ).